Sources/ClientRuntime/Telemetry/DefaultTelemetry.swift

```diff
@@ -0,0 +1,151 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Namespace for the Default SDK Telemetry implementations.
+public enum DefaultTelemetry {
+    /// The Default SDK Telemetry Provider Implementation.
+    ///
+    /// - contextManager: no-op
+    /// - loggerProvider: provides SwiftLoggers
+    /// - meterProvider: no-op
+    /// - tracerProvider: no-op
+    public static let Provider: TelemetryProvider = _DefaultTelemetryProvider()
+
+    fileprivate class _DefaultTelemetryProvider: TelemetryProvider {
+        let contextManager: TelemetryContextManager = DefaultTelemetryContextManager
+        let loggerProvider: LoggerProvider = DefaultLoggerProvider
+        let meterProvider: MeterProvider = DefaultMeterProvider
+        let tracerProvider: TracerProvider = DefaultTracerProvider
+    }
+}
+
+// Context
+extension DefaultTelemetry {
+    fileprivate static let DefaultTelemetryContextManager: TelemetryContextManager = NoOpTelemetryContextManager()
+    fileprivate static let DefaultTelemetryContext: TelemetryContext = NoOpTelemetryContext()
+    fileprivate static let DefaultTelemetryScope: TelemetryScope = NoOpTelemetryScope()
+
+    fileprivate class NoOpTelemetryContextManager: TelemetryContextManager {
+        func current() -> TelemetryContext { DefaultTelemetryContext }
+    }
+    fileprivate class NoOpTelemetryContext: TelemetryContext {
+        func makeCurrent() -> TelemetryScope { DefaultTelemetryScope }
+    }
+    fileprivate class NoOpTelemetryScope: TelemetryScope {
+        func end() {}
+    }
+}
+
+// Logging
+extension DefaultTelemetry {
+    fileprivate static let DefaultLoggerProvider: LoggerProvider = _DefaultLoggerProvider()
+
+    fileprivate class _DefaultLoggerProvider: LoggerProvider {
+        func getLogger(name: String) -> LogAgent { SwiftLogger(label: name) }
+    }
+}
+
+// Metrics
+extension DefaultTelemetry {
+    fileprivate static let DefaultMeterProvider: MeterProvider = NoOpMeterProvider()
+    fileprivate static let DefaultMeter: Meter = NoOpMeter()
+    fileprivate static let DefaultAsyncMeasurementHandle: AsyncMeasurementHandle = NoOpAsyncMeasurementHandle()
+    fileprivate static let DefaultUpDownCounter: UpDownCounter = NoOpUpDownCounter()
+    fileprivate static let DefaultMonotonicCounter: MonotonicCounter = NoOpMonotonicCounter()
+    fileprivate static let DefaultHistogram: Histogram = NoOpHistogram()
+
+    fileprivate class NoOpMeterProvider: MeterProvider {
+        func getMeter(scope: String, attributes: Attributes?) -> Meter { DefaultMeter }
+    }
+    fileprivate class NoOpMeter: Meter {
+        func createGauge(
+            name: String,
+            callback: @escaping (any DoubleAsyncMeasurement) -> Void,
+            units: String?,
+            description: String?
+        ) -> AsyncMeasurementHandle {
+            DefaultAsyncMeasurementHandle
```

Comment:
Not needed to keep static values around.  Can just create & return a no-op instance when requested.